old-src/endpoints/v1.es6.js

```diff
@@ -1239,7 +1239,10 @@ class APIv1Endpoint {
   get messages () {
     return bindAll({
       buildOptions: function(options) {
-        const uri = options.origin + '/message/' + (options.view || 'inbox');
+        const { filter, subredditName, origin, view } = options;
```

Comment:
you can default `view` here:

`const { filter, subredditName, origin, view = 'inbox' } = options;`